dev-docs/bidders/brave.md

```diff
@@ -11,7 +11,10 @@ media_types: banner, video, native
 safeframes_ok: true
 deals_supported: true
 pbjs: true
-pbs: false
+pbs: true
+gvl_id: 869
+floors_supported: true
+pbs_app_supported: true
```

Comment:
While we're waiting on the code to be merged, I'd like to ask that you guys flesh out the rest of the documentation metadata as defined in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter . You're missing:
```
fpd_supported (but can't set this to true without details in the body below)
dchain_supported
deals_supported
ortb_blocking_supported
userId
```